libretro/libretro.cpp

```diff
@@ -347,6 +348,15 @@ static void check_variables(CoreParameter &coreParam)
    if (g_Config.iLanguage < 0)
       g_Config.iLanguage = get_language_auto();
 
+   g_Config.sLanguageIni = "en_US";
+   auto langValuesMapping = GetLangValuesMapping();
+   for(auto i = langValuesMapping.begin(); i != langValuesMapping.end(); ++i){
+      if(i->second.second == g_Config.iLanguage){
+         g_Config.sLanguageIni = i->first;
+	  }
+   }
```

Comment:
can you fix the code formatting to match the surrounding code? specifically with regards to tabs, and spaces here and there.

Reply:
Wow what a quick response and apologies for rude. Please point if other suggestions.

Reply:
Thanks! Code looks good to me.